src/it/resources/com/google/checkstyle/test/chapter4formatting/rule413emptyblocks/InputFormattedEmptyBlocksAndCatchBlocks.java

```diff
@@ -4,19 +4,19 @@ class InputFormattedEmptyBlocksAndCatchBlocks {
   static {
   }
 
+  static {
+  }
+
+  static {
+  }
+
```

Comment:
We already have a `static` block above. Why do we need 3 static blocks in a row?

Reply:
It is Formatted file, it better to have it to minimize diff with original Input file

Reply:
> We already have a static block above. Why do we need 3 static blocks in a row?

in OG file, these blocks had different `{}` placements, in formatted file formatter made it look same